pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1321,19 +1304,15 @@ public void updateIndexingConfigFor(String tableName, TableType type, IndexingCo
     setExistingTableConfig(tableConfig);
   }
 
-  private void handleBrokerResource(@Nonnull final String tableName, @Nonnull final List<String> brokersForTenant) {
-    LOGGER.info("Updating BrokerResource IdealState for table: {}", tableName);
-    HelixHelper
-        .updateIdealState(_helixZkManager, Helix.BROKER_RESOURCE_INSTANCE, new Function<IdealState, IdealState>() {
-          @Override
-          public IdealState apply(@Nullable IdealState idealState) {
-            Preconditions.checkNotNull(idealState);
-            for (String broker : brokersForTenant) {
-              idealState.setPartitionState(tableName, broker, BrokerOnlineOfflineStateModel.ONLINE);
-            }
-            return idealState;
-          }
-        }, RetryPolicies.exponentialBackoffRetryPolicy(5, 500L, 2.0f));
+  private void handleBrokerResource(String tableNameWithType, List<String> brokersForTenant) {
```

Comment:
suggest rename to enableTableInBrokers(). Or, updateBrokerResource() but that is more vague.

Reply:
Inline it into the addTable() method to be more clear